### DISCUSSION TITLE
Section names should not be unique

### DISCUSSION THREAD

**Discussion by mini71** [craftcms/cms#9829] [Ideas] [answered]
In a multi site setup it would be helpful to create structure/channel sections with the same name (e,g, "Pages").

If the entry types (+fields) differ between sites it is currently necessary to create seperate sections. From a content editor perspective they should have the same display name.

Since section handles are already the unique key, it does not make sense to validate the names for uniqueness too

**Comment by TheFunkyMonk** [comment]
Ran into this today with a group of sections with duplicative names but unique handles:

![2024-10-01 at 15 50 30](https://github.com/user-attachments/assets/8c825a73-3e7b-4b65-a12c-ca474a4abb3e)

I was able to work around it using a hidden unicode character after the title in the section name, but would be nice to have this ability since it's already possible with Entry Types and Fields.

**Comment by brandonkelly** [comment, marked as answer]
Just made it possible to create sections with non-unique names for Craft 5.5 (88587ca2f505c083ef2dd18f0252f1038b4bffd4).

**Reply by brandonkelly** [reply]
Craft 5.5.0 is out with that change 🎉
